app/javascript/components/payment-error-button.vue

```diff
@@ -116,7 +117,21 @@ export default {
       this.showCartOnDebtModal = false;
       this.showNameTextBox = false;
       this.showDebtFinalMessage = true;
+      const products = this.cartProductsToReqFormat();
+      console.log(this.cartProductsToReqFormat);
+      invoiceApi.createDebtProduct({
+        "debtor": this.message,
+        "products": products,
+       });
     },
+      cartProductsToReqFormat() {
+        const productsArray = this.products.filter(product => product.amount > 0);
+        let productsToReqFormatArray = [];
+        productsArray.forEach(prod => {
+          productsToReqFormatArray.push({ "product_id": prod.id, "product_price": prod.price });
+        });
+        return productsToReqFormatArray;
```

Comment:
Expected indentation of 6 spaces but found 8                         indent<br>Expected newline before return statement                             newline-before-return